app/models/concerns/organizationable.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+###
+# @description: This module defines methods for models that are
+#   related to organizations
+###
+module Organizationable
```

Comment:
Let's add a `vocabularies` association to `Organization` instead. That'll simplify creating and finding them:

```ruby
class Organization < ApplicationRecord
  ...
  has_many :vocabularies
  ...
end
```
```ruby
def index
  vocabularies = current_user.organization.vocabularies.order(:name)
  ...
end

def create
  vocabulary = current_user.organization.vocabularies.create!(valid_params) # no need to merge the organization into the params anymore
  ...
end
```

Reply:
OK, I like that.